torque-core/src/main/kotlin/com/workday/torque/ModuleTestParser.kt

```diff
@@ -30,12 +31,11 @@ class ModuleTestParser(private val args: Args, private val apkTestParser: ApkTes
 			includedAnnotations: List<String>,
 			excludedAnnotations: List<String>
     ): Boolean {
-        return annotations.all { includedAnnotations.hasAnnotation(it.name) }
-                && excludedAnnotations.all { includedAnnotations.hasAnnotation(it).not() }
+        return includedAnnotations.all { annotations.hasAnnotation(it) } && excludedAnnotations.all { annotations.hasAnnotation(it).not() }
```

Comment:
Nice find! This seems like it should've been caught by tests before. Are the unit tests not being run by CI or do we not have tests covering this particular logic?

Reply:
its spek, I can add a test to catch this

Reply:
oh man 🙈

Reply:
hmm did we never use exclusions before? Maybe it was a YAGNI situation hmm...
🙈

Reply:
oh it wasn't me lol
but yeah curious why the test didn't trigger the fail

Reply:
yeah this was my bad, the extensions actually got confusing when the types changed from dex test parser